EG23.tex

```diff
@@ -180,30 +195,35 @@ \subsection{Reactors}
 %% Apropos my comment above - this is a great example of showing others how to
 %% translate specifications into Cyclus inputs.
 
-Because of the 1-month timestep of Cyclus, 
-the batch quantity and cycle length parameters have been manipulated to match as
-closely as possible important invariants such as fuel residence time, burnup,
-and discharge rates \cite{B.Feng_calculation}.  For the SFR 1 facility type, the cycle
-length is specified to be 5.44 y (for discharge burnup of 37.62 GWd/t), 5.41 y
-has been used (5 $\times$ 13 months). For SFR 2, 4.5y has been used (3 $\times$ 18 months)
-where 4.44 was expected.  With Cyclus' 1-month time step, the burnup resolution
-can be easly calculated as:
-
-\begin{equation}
-  \Delta BU = \frac{P_{th} \times \Delta t}{M_{r}},
-\end{equation}
-
-\noindent where $P_{th}$ is the thermal power of the reactor, $M_{r}$ its mass
-in heavy metal, $\Delta BU$ the burnup achievable precision and $\Delta t$ the
-time step size.  In this simulation the $\Delta BU$ are for the SFR 1 \& 2
-respectively: $0.73$ and $0.79~GWd/t$.  This does not affect the composition
-of the discharge fuel as the reactor model used for this analysis does not
-perform a burnup evolution calculation.  Only the mass of discharged fuel is
-affected with a relative error of :
+Because of the 1-month timestep of Cyclus, the batch quantity and cycle length
+parameters have been manipulated to match as closely as possible important
+invariants such as fuel residence time, burnup, and discharge rates
+\cite{B.Feng_calculation}.
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% NOTE: I am note sure it birng any interresing information, so I remove it and I
+%% add the actual error made on the mass for each reactor.
+
+% For the SFR 1 facility type, the cycle
+%length is specified to be 5.44 y (for discharge burnup of 37.62 GWd/t), 5.41 y
+%has been used (5 $\times$ 13 months). For SFR 2, 4.5y has been used (3 $\times$ 18 months)
+%where 4.44 was expected.  With Cyclus' 1-month time step, the burnup resolution
+%can be easly calculated as:
+%\begin{equation}
+%  \Delta BU = \frac{P_{th} \times \Delta t}{M_{r}},
+%\end{equation}
 
-\begin{equation}
-  \frac{\Delta M}{M} = \frac{\Delta BU}{BU}.
-\end{equation}
+%\noindent where $P_{th}$ is the thermal power of the reactor, $M_{r}$ its mass
+%in heavy metal, $\Delta BU$ the burnup achievable precision and $\Delta t$ the
+%time step size. In this simulation the $\Delta BU$ are for the SFR 1 \& 2
+%respectively: $0.73$ and $0.79~GWd/t$. 
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+This does not affect the composition
+of the discharge fuel as the reactor model used for this analysis does not
+perform a burnup evolution calculation. Only the mass of discharged fuel is
+affected with a respective relative error : $0.01 \%$, $0.0 \%$ and $0.006 \%$. 
 
 %% NOTE: This ends abruptly... 
```

Comment:
> NOTE: This ends abruptly... 
>    do we calculate this for the different reactors?
>    do we know what it is for the LWR since we don't know the efficiency?

I have modify a bit this part since the actual BU error was not very interesting... I have added the actual values on the relative mass error for each reactor, which make more sense...